system/RESTful/ResourcePresenter.php

```diff
@@ -88,18 +88,16 @@ public function index()
 	/**
 	 * Present a view to present a specific resource object
 	 *
-	 * @param  type $id
 	 * @return string
 	 */
-	public function show($id = null)
```

Comment:
All methods in ResourcePresenter with the `$id=null` arguments must stay. It's intended to be extended and doesn't work if they're missing.